x-pack/metricbeat/module/gcp/metrics/metrics_requester.go

```diff
@@ -198,6 +211,7 @@ func (r *metricsRequester) getFilterForMetric(serviceName, m string) string {
 
 		switch {
 		case r.config.Region != "":
+			// FIXME: using resource.labels.zone but should use region.
```

Comment:
I though about this, let's remove it from here and do it for good separately. I'll open an issue to discuss how to address this.
```suggestion
```